src/cco-modules/GeospatialOntology.ttl

```diff
@@ -55,6 +56,7 @@ cco:connected_with rdf:type owl:ObjectProperty ;
                    rdfs:domain obo:BFO_0000141 ;
                    rdfs:range obo:BFO_0000141 ;
                    cco:definition "An immaterial entity im1 is connected with some immaterial entity im2 iff there exists some immaterial entity im3 that is common to both im1 and im2."@en ;
```

Comment:
This seems closer to 'overlaps' in the cited document. Connection is just reflexive and symmetric.